src/super_gradients/training/__init__.py

```diff
@@ -1,11 +1,12 @@
 # PACKAGE IMPORTS FOR EXTERNAL USAGE
 import super_gradients.training.utils.distributed_training_utils as distributed_training_utils
-from super_gradients.training.datasets import datasets_utils, DataAugmentation, TestDatasetInterface, SegmentationTestDatasetInterface,\
+from super_gradients.training.datasets import datasets_utils, DataAugmentation, TestDatasetInterface, SegmentationTestDatasetInterface, \
     DetectionTestDatasetInterface, ClassificationTestDatasetInterface
 from super_gradients.training.models import ARCHITECTURES
-from super_gradients.training.sg_model import SgModel, MultiGPUMode, StrictLoad
-from super_gradients.training.kd_model import KDModel
+from super_gradients.training.sg_trainer import Trainer, \
+    MultiGPUMode, StrictLoad
+from super_gradients.training.kd_trainer import KDTrainer
 
```

Comment:
We need to keep all of these imports, just make them point to the deprecated class. Otherwise we'd break user's code if this is how they imported it.

Reply:
done. fixed imports

Reply:
still missing as far as I can see.